src/main/java/core/basesyntax/service/writer/WriteFileFruitImpl.java

```
@@ -0,0 +1,16 @@
+package core.basesyntax.service.writer;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteFileFruitImpl implements WriteFileFruit {
+    @Override
+    public void writeAll(String string, String path) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(path))) {
+            bufferedWriter.write(string);
+        } catch (IOException e) {
+            System.out.println("error: file not found." + e);

```

Current catch block prints the exception to stdout: `System.out.println("error: file not found." + e);`. This violates the checklist item: "Don't ignore exceptions in try-catch blocks". The checklist shows the preferred approach: `throw new RuntimeException("Can't find file by path: " + filePath, e);`. Replace the println with rethrowing a RuntimeException that includes the `path` and passes `e` as the cause so callers/tests can observe the failure.